app/Doctors/doctorsFound.tsx

```diff
@@ -0,0 +1,252 @@
+      images: require("../../assets/doctors/doc2.png"),
+    },
+    {
+      name: "Dr. Jenny Watson",
+      role: "Immunologists",
+      hospital:"Christ Hospital",
+      image: require("../../assets/doctors/like.png"),
+      stars: "4.4",
+     reviews:"4,942",
+      images: require("../../assets/doctors/doc3.png"),
+    },
+    {
+      name: "Dr. Raul Zirkind",
+      role: "Neurologists",
+      hospital:"Franklin Hospital",
+      image: require("../../assets/doctors/like.png"),
+      stars: "4.8",
+      reviews:"6,362",
+      images: require("../../assets/doctors/doc1.png"),
+    },
+    {
+      name: "Dr. Elijah Baranick",
+      role: "Allergists",
+      hospital:"JFK Medical Center",
+      image: require("../../assets/doctors/like.png"),
+      stars: "4.6",
+      reviews:"5,366",
+      images: require("../../assets/doctors/doc2.png"),
+    },
+    {
+      name: "Dr. Stephen Shute",
+      role: "Cardiologist",
+      hospital:"The valley Hospital",
+      image: require("../../assets/doctors/like.png"),
+      stars: "4.8",
+      reviews:"3,279",
+      images: require("../../assets/doctors/doc5.png"),
+    },
+    
+  ];
+
+  return (
+    <ScrollView>
+    <View className="flex-1 bg-white px-4 py-5">
+       <View className="pt-10">
+     
+       <View className="flex-row"> 
+       <NavigationHeader title=""/>
+                  <SearchInput/>
+                  </View>
+        
+    <DocButtons/>
+    </View>
+    <View style={styles.found}>
+        <Text style={styles.btnTxt}>489 founds</Text>
+        <View className="flex-row">
+    <Text style={styles.default} onPress={toggleModal}>Default  </Text>
+     <Image source={require("../../assets/doctors/arrows.png")}/>
+     </View>
+        </View>
+        {docCards.map((spot, index) => (
+              <DoctorCard key={index} {...spot} />
+        ))}
+       <Modal
+          visible={isVisible}
+          animationType="fade"
+          transparent={true}
+          onRequestClose={toggleModal}
+        >
+          <View style={{ flex: 1, justifyContent: 'flex-end', alignItems: 'flex-end', backgroundColor: 'rgba(0,0,0,0.5)' }}>
+            <View style={{ backgroundColor: 'white', padding: 5, borderTopRightRadius: 48, borderTopLeftRadius: 48 }}>
+            <Text style={styles.title2}>Filter</Text>
+    <Text style={styles.line}></Text>
+    <View>
+    <Text style={styles.subtitles}>Speciality
+    </Text>
+    <DocButtons/>
+    <Text style={styles.subtitles}>Rating
+    </Text>
+ <ReviewButtons/>
+ <Text style={styles.line}></Text>
+ </View>
+ <View style={styles.btns}>
+ <TouchableOpacity style={styles.regularBn}>
+    <Text style={styles.buttonText} onPress={() => {
+                router.push("Doctors/doctorsFound")}}>Reset</Text>
+</TouchableOpacity>
+<TouchableOpacity style={styles.activeBn}>
+    <Text style={styles.activeTxt} onPress={() => {
+                router.push("Doctors/doctorsFound")}}>Apply</Text>
+</TouchableOpacity>
```

Comment:
These button should hide the modal by setting the `isVisible` state not routing to the same page. When clicked they go to the same page but the modal doesn't close.

Reply:
Thank you let me work on that